from test_preconditions.exceptions import mapi_exceptions